spec/pusher/push_notifications/generate_token_spec.rb

```diff
@@ -3,51 +3,49 @@
 require 'spec_helper'
 require 'jwt'
 
-RSpec.describe Pusher::PushNotifications::UseCases::GenerateToken do
-  subject(:use_case) { described_class.new(user: user) }
+RSpec.describe Pusher::PushNotifications, '.generate_token' do
+  def generate_token
+    described_class.generate_token(user: user)
```

Comment:
Test use cases through the `PushNotifications` model in order to exercise the delegation